script/foreman-debug

```diff
@@ -286,6 +286,7 @@ if [ $NOGENERIC -eq 0 ]; then
   add_files /var/log/messages*
   add_files /var/log/audit/audit.log*
   add_files /var/log/syslog*
+  add_cmd "getenforce" "selinux_enforce"
```

Comment:
not necessary but IMHO `selinux_state` file would be better.

```sh
[root@katello ~]# getenforce 
Permissive

Reply:
Done thanks